cdap-examples/UserProfiles/bin/add-users.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+#
+# Copyright © 2014 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+bin=`dirname "${BASH_SOURCE-$0}"`
+bin=`cd "$bin"; pwd`
+script=`basename $0`
+
+auth_token=
+auth_file="$HOME/.cdap.accesstoken"
+verbose=false
+
+function get_auth_token() {
+  if [ -f $auth_file ]; then
+    auth_token=`cat $auth_file`
+  fi
+}
+
+function usage() {
+  echo "Tool for adding users to the profiles table."
```

Comment:
Because the codes are almost identical, and less code to maintain == fewer headaches...but, this being an example, clarity is more important than brevity. Your call.